### ISSUE TITLE
serial command 'screen()' ValueError

### ISSUE THREAD

**Issue by thcasey3** [Bridge12Technologies/pyB12MPS#4]
pyB12MPS v1.0.6
Firmware v1.5.3
```
mps.screen() 
```
returns:
```
ValueError: invalid literal for int() with base 10: 'ERROR'
```

**Comment by tkellerBridge12** [comment]
This is coming from two different versions of the firmware which have different serial commands to query the screen status. In the old version of the firmware (v1.4) "screenstatus?" was sent. In the new version of the firmware "screen?" is the new command.

A current workaround would be:
`send_command('screen?', recv = True)`

I'll think about a way to update the code so it is compatible with both versions of the firmware.

**Comment by thorstenmaly** [comment]
Resolved in 1.0.8
